pkg/kubectl/resource/visitor.go

```diff
@@ -65,6 +65,7 @@ type Info struct {
 	Mapping   *meta.RESTMapping
 	Namespace string
 	Name      string
+	Source    string
```

Comment:
Please add a comment explaining what "Source" means. Is this the name of a .go file? a .json/.yaml file?